docs/references.bib

```diff
@@ -6,3 +6,101 @@ @article{IEEE754
   pages={1-84},
   doi={10.1109/IEEESTD.2019.8766229}
 }
+
+@article{BansalBRT17,
+  author    = {Kshitij Bansal and
+               Clark W. Barrett and
+               Andrew Reynolds and
+               Cesare Tinelli},
+  title     = {A New Decision Procedure for Finite Sets and Cardinality Constraints
+               in {SMT}},
+  journal   = {CoRR},
+  volume    = {abs/1702.06259},
+  year      = {2017},
+  url       = {http://arxiv.org/abs/1702.06259},
+  archivePrefix = {arXiv},
+  eprint    = {1702.06259},
+  timestamp = {Mon, 13 Aug 2018 16:47:11 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/BansalBRT17.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+@inproceedings{MengRTB17,
+  author    = {Baoluo Meng and
+               Andrew Reynolds and
+               Cesare Tinelli and
+               Clark W. Barrett},
+  editor    = {Leonardo de Moura},
+  title     = {Relational Constraint Solving in {SMT}},
+  booktitle = {Automated Deduction - {CADE} 26 - 26th International Conference on
+               Automated Deduction, Gothenburg, Sweden, August 6-11, 2017, Proceedings},
+  series    = {Lecture Notes in Computer Science},
+  volume    = {10395},
+  pages     = {148--165},
+  publisher = {Springer},
+  year      = {2017},
+  url       = {https://doi.org/10.1007/978-3-319-63046-5\_10},
+  doi       = {10.1007/978-3-319-63046-5\_10},
+  timestamp = {Wed, 25 Sep 2019 18:19:14 +0200},
+  biburl    = {https://dblp.org/rec/conf/cade/MengRTB17.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+@inproceedings{ReynoldsIS016,
```

Comment:
Any reason why this particular reference is missing the fourth author in the name?

Reply:
No idea. That's how it is in dblp. I can change it.